src/Deprecations.jl

```diff
@@ -89,3 +89,6 @@ end
 @deprecate is_power(x::IntegerUnion) is_perfect_power_with_data(x)
 @deprecate is_power(x::QQFieldElem) is_perfect_power_with_data(x)
 @deprecate is_power(x::Rational) is_perfect_power_with_data(x)
+
+@deprecate coefficient_ring(k::PadicField) base_field(k)
+@deprecate coefficient_ring(k::QadicField) base_field(k)
```

Comment:
```suggestion
# To deprecate in 0.47
coefficient_ring(k::PadicField) = base_field(k)
coefficient_ring(k::QadicField) = base_field(k)
```
couldn't we change this deprecation like this for now and get this merged? If I understand correctly, this should be non-breaking, right?

Reply:
No. There is a mess in Hecke with `base_field` being defined via `coefficient_ring`, so this breaks Hecke either way. I am currently preparing a pull request in Hecke that goes with this one.